HT-2.py

```diff
@@ -0,0 +1,67 @@
+# Підраховує кількість голосних літер у тексті
+
+
+def count_vowels(text):
+    vowels = 'AEIOUaeiou'
+    vowel_counts = {vowel: 0 for vowel in vowels}
+    for char in text:
+        if char in vowels:
+            vowel_counts[char] += 1
+
+    return vowel_counts
+
+# При запуску цього коду з використанням наданого тексту,
+# він виведе кількість голосних літер у форматі таблиці,
+# з окремими підрахунками для голосних літер у нижньому і верхньому регістрах.
+
+
+def print_vowel_counts(vowel_counts):
+    print("-" * 25)
+    print(f"| {'vowel':^10} | {'count':^8} |")
+    print("-" * 25)
+    for vowel, count in vowel_counts.items():
+        print(f"| {vowel:^10}| {count:^10.0f}|")
+    print("-" * 25)
+
+
+poem_text = """I wandered lonely as a cloud
+That floats on high o'er vales and hills,
+When all at once I saw a crowd,
+A host, of golden daffodils;
+Beside the lake, beneath the trees,
+Fluttering and dancing in the breeze.
+
+Continuous as the stars that shine
+And twinkle on the Milky Way,
+They stretched in never-ending line
+Along the margin of a bay:
+Ten thousand saw I at a glance,
+Tossing their heads in sprightly dance."""
+
+vowel_counts = count_vowels(poem_text)
+print_vowel_counts(vowel_counts)
+
+print(' ' * 20)
+
+
+def modify_text(text):
```

Comment:
Too complicated. Here could be recommended two approaches:
1. replace aka replace_text = poem_text.replace('A', 'À').replace('a', 'à')
2. More simple way -> use translate tables with two string methods
change_strings = str.maketrans('AEIOUaeiou', 'ÀÉÍÓÚàéíóú')
modified_poem = poem.translate(change_strings)